core/services/ocr2/plugins/ocr2keeper/evmregistry/v21/mercury/v03/request.go

```diff
@@ -180,6 +182,7 @@ func (c *client) multiFeedsRequest(ctx context.Context, ch chan<- mercury.Mercur
 			}
 
 			c.lggr.Infof("at timestamp %s upkeep %s received status code %d from mercury v0.3", sl.Time.String(), sl.UpkeepId.String(), resp.StatusCode)
+			prommetrics.AutomationStreamsResponses.WithLabelValues(prommetrics.StreamsVersion03, http.StatusText(resp.StatusCode)).Inc()
```

Comment:
Let's go with `fmt.Sprintf("%d", resp.StatusCode)` instead of `http.StatusText(resp.StatusCode)` to have the status code as string rather than text (e.g. "not found")